### DISCUSSION TITLE
With React Native Web Example with iOS/Android Setups

### DISCUSSION THREAD

**Discussion by tettoffensive** [vercel/next.js#43387] [Help]
# Feature request

Would love it if the example for using Next.js with React Native Web also included React Native iOS/Android setup. Just need to better understand the file structure setup. Since, both these platforms point to a different entry point than Next.js uses.

https://github.com/zeit/next.js/tree/master/examples/with-react-native-web

**Comment by sotomaque** [comment]
following up on this. any documentation on going from the above link to running the app on a simulator?

**Comment by jankaifer** [comment]
We are tracking feature requests in discussions so I'll move this.

We would love to review your PR if you want to contribute these examples.
